upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -152,7 +156,49 @@ func (c *ApplyClusterCmd) Run() error {
 	}
 	c.channel = channel
 
-	assetBuilder := assets.NewAssetBuilder(c.Cluster.Spec.Assets)
+	stageAssetsLifecycle := fi.LifecycleSync
```

Comment:
Had to move the lifecycle code higher, because we need to use phase in the assets builder.